Extensions/BBTextLabel/JsExtension.js

```diff
@@ -0,0 +1,385 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+
+//TODO: add base color, base font, base size
+const MultiStyleText = require('./dist/pixi-multistyle-text.umd');
+console.log(MultiStyleText);
+module.exports = {
+  createExtension: function(_, gd) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        'BBText',
+        'BBText',
+        _('Display a BBText object on the scene.'),
+        'Todor Imreorov',
+        'Open source (MIT License)'
+      )
+      .setExtensionHelpPath('/objects/BBText');
+
+    var ObjectBBText = new gd.ObjectJsImplementation();
+    ObjectBBText.updateProperty = function(
+      objectContent,
+      propertyName,
+      newValue
+    ) {
+      if (propertyName === 'Text') {
```

Comment:
nitpicking again: for consistency with the way member variables are written in C++ and JS (all lowercase), name your properties in lowercase ('text', 'color', etc...).
PascalCase should be only for objects and, in general, things that are akin to "classes" in programming languages. (which is the case for an extension/object/behavior, but not for a property)

Reply:
I lifted that from the video extension, thinking GD expects it for some reason :)